CDP4Common/Poco/RelationalOperatorKindMethods.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace CDP4Common.EngineeringModelData
+{
+    /// <summary>
+    /// Extension methods for the <see cref="RelationalOperatorKind"/> enum
+    /// </summary>
+    public static class RelationalOperatorKindMethods
+    {
+        /// <summary>
+        /// Extension method that converts a <see cref="RelationalOperatorKind"/> to a readable string
+        /// </summary>
+        /// <param name="value"><see cref="RelationalOperatorKind"/> to convert</param>
+        /// <returns></returns>
+        public static string ToFriendlyString(this RelationalOperatorKind value)

```

Comment:
not so happy with the name of the method... why not just `ToString()` ?

Reply:
ToString was also my first idea, but all objects already have a ToString method. There could be side effects, now and in the future.

ToScientificString?
Other ideas?

Reply:
I tend to agree that overriding `ToString()` of enums might have some unwanted circumstances.

I would propose `ToHumanReadableString()`, long but says exactly what it does.

Reply:
Lets do `ToScientificNotationString()`.